crates/router/src/utils/user_role.rs

```diff
@@ -72,31 +72,21 @@ pub async fn set_role_permissions_in_cache_by_user_role(
     state: &SessionState,
     user_role: &UserRole,
 ) -> bool {
-    let Some(ref merchant_id) = user_role.merchant_id else {
-        return false;
-    };
-
     let Some(ref org_id) = user_role.org_id else {
         return false;
     };
-    set_role_permissions_in_cache_if_required(
-        state,
-        user_role.role_id.as_str(),
-        merchant_id,
-        org_id,
-    )
-    .await
-    .map_err(|e| logger::error!("Error setting permissions in cache {:?}", e))
-    .is_ok()
+    set_role_permissions_in_cache_if_required(state, user_role.role_id.as_str(), org_id)
+        .await
+        .map_err(|e| logger::error!("Error setting permissions in cache {:?}", e))
+        .is_ok()
 }
 
-pub async fn set_role_permissions_in_cache_by_role_id_merchant_id_org_id(
+pub async fn set_role_permissions_in_cache_by_role_id_org_id(
```

Comment:
Can we rename these functions to `set_role_info...` instead of `set_permissions...`.